managed/services/telemetry/config.default.yml

```diff
@@ -57,6 +57,14 @@ telemetry:
       - metric_name: "mysql_version"
         label: "version"
 
+  - id: MySQLVersionComment
+    source: VM
+    query: mysql_version_info{service_type="mysql"}
+    summary: "Detailed information about version of monitored MySQL services"
+    data:
+      - metric_name: "mysql_version_comment"
+        label: "version_comment"
```

Comment:
it can be added to the prev block (MySQLVersion), so it will look like:
```
data:
  - metric_name: "mysql_version"
    label: "version"
  - metric_name: "mysql_version_comment"
    label: "version_comment"
```

In such a case there will be only one query execution to VM instead of 2

Reply:
But, in this case, I will not be able to have separate descriptions for metrics?

Reply:
you can use the following desc:

“Detailed information about version and comment of monitored MySQL services”